Server/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -174,13 +174,15 @@ void CLuaPedDefs::AddClass(lua_State* luaVM)
 
 int CLuaPedDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
-    lua_newtable(luaVM);
-    for (int i = 0; i <= 312; i++)
+    // Out of these 312, 299 are valid, so reserve that much
+    lua_createtable(luaVM, 299, 0);  
```

Comment:
```suggestion
    lua_createtable(luaVM, 314, 0);  
```
Let's do 313 or 314, just to be safe.

Reply:
No need to. There aren't 314 skins actually (that's the highest skinID tho).